README.md

```diff
@@ -0,0 +1,136 @@
+
+# First-Capstone
+=======
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Capstone-Project: Summer BootCamp 2023 ](#-capstone-project-summer-bootcamp-2023-)
+  - [🛠 Built With ](#-built-with-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Capstone-Project: Summer BootCamp 2023 <a name="about-project"></a>
+
+**[Capstone-Project: Summer BootCamp 2023]** is about crash courses in Computer Science during Summer. People can visit the website to see the details of the program.
+
+[Loom Video Presentation](https://www.loom.com/share/8265989651e04953b74b70498aef592e)
+
+## 🛠 Built With <a name="built-with"></a>
+    HTML
+    CSS
+    Javascript
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Summer BootCamp 2023]( https://salwa99.github.io/First-Capstone/)
```

Comment:
- [ ] Unfortunately, your live demo is not working. Please fix the link and post the correct link of your app deployed on Github Pages.